test/refda/test_reporting.c

```diff
@@ -72,6 +74,9 @@ static void test_reporting_edd_one_int(const refda_amm_edd_desc_t *obj _U_, refd
 void setUp(void)
 {
     refda_agent_init(&agent);
+    cace_data_init(&mgr);
+    cace_data_copy_from(&mgr, 4, (cace_data_ptr_t)"test");
```

Comment:
**Nitpick**: Could consider using `strlen` to derive length instead of hardcoding here.